### ISSUE TITLE
Increased bundle size when using react-intl

### ISSUE THREAD

**Issue by menelike** [meteor/meteor#9097]
- tested on `METEOR@1.5.1` and `METEOR@1.5.2`
- mac OS Sierra 10.12.6 (16G29)

When using `react-Intl` the bundle size increases from initially **701 kb** to **1.45 mb** in the simple todo app.

**before: https://github.com/menelike/meteorIntl/commit/f1fc56a877458c099be120982b794fd6463b97f6**
![bildschirmfoto 2017-09-16 um 08 04 23](https://user-images.githubusercontent.com/4238881/30509713-499f5efa-9ab7-11e7-9add-b37874c83f65.png)

**after: https://github.com/menelike/meteorIntl/commit/7daf96aad653a9fdc2a5343da6b819eaf3f225a8**
![bildschirmfoto 2017-09-16 um 08 08 21](https://user-images.githubusercontent.com/4238881/30509714-4c3a4b0c-9ab7-11e7-8062-48d68891fbb8.png)

The new bundle has included all available languages from `react-intl` and `intl-relativeformat`.
![bildschirmfoto 2017-09-16 um 08 08 30](https://user-images.githubusercontent.com/4238881/30509756-0e773054-9ab8-11e7-8352-c350717f0ad6.png)
![bildschirmfoto 2017-09-16 um 08 08 38](https://user-images.githubusercontent.com/4238881/30509757-0e7fbcc4-9ab8-11e7-8013-0bbf1c6fb3cd.png)

So why is that?

It think that Meteor does not handle the `browser` flag in `package.json` when determining the files which should be excluded from the bundle https://github.com/defunctzombie/package-browser-field-spec#ignore-a-module.

*Keep in mind that this is the desired behavior on server side* 

`package.json` from `intl-relativeformat`
```
  "browser": {
    "./lib/locales": false,
    "./lib/locales.js": false
  },
```
https://github.com/yahoo/intl-relativeformat/blob/7288de4fe04141b3a73613408e0eee81470ba567/package.json#L26-L29

As a result this import which should have no impact in the browser, will include all available languages.
```
// Add all locale data to `IntlRelativeFormat`. This module will be ignored when
// bundling for the browser with Browserify/Webpack.
require('./lib/locales');
```
https://github.com/yahoo/intl-relativeformat/blob/7288de4fe04141b3a73613408e0eee81470ba567/index.js#L7-L9

So far I've not seen any other solution than starting a fork like discussed here https://forums.meteor.com/t/i18n-react-intl-importing-only-locales-you-need/36970/3.

In our project this problem occurs with several other packages like `fabricjs` too, so forking is not an option.

I highly assume that a project under those circumstances will have a massively decreased performance on client (network load, first render as unnecessary imports need to be parsed as well) 

Thanks a lot in advance!

**Comment by klaussner** [comment]
Thanks for the detailed description! :slightly_smiling_face: This looks like a duplicate of #6890, though.

**Comment by menelike** [comment]
I'm sorry for the duplicate, thanks a lot for the reference!
